classes/Carrier.php

```diff
@@ -72,7 +72,7 @@ class CarrierCore extends ObjectModel
     /** @var bool True if extra shipping handling cost should be applied to this Carrier */
     public $shipping_handling = true;
 
-    /** @var bool Behavior for out-of-range weights: true to disable carrier, false to apply the cost of the highest defined range */
+    /** @var bool|int Behavior for out-of-range weights: true to disable carrier, false to apply the cost of the highest defined range */
```

Comment:
I don't think we should change this So far the value is still a `tinyint(1)` so a boolean It's good if we improve this in the CQRS commands/queries but modifying the legacy class doesn't seem relevant

Reply:
I guess it's not happy when you integrate the boolean from the ObjectModel into the QueryResult that uses an integer right? Then you can force casting the boolean into an int, this way you don't need to modify the ObjectModel

Reply:
This is still not necessary